src/build.h

```diff
@@ -190,7 +190,7 @@ struct Builder {
 
   /// Add a target to the build, scanning dependencies.
   /// @return false on error.
-  bool AddTarget(Node* target, string* err);
+  bool AddTarget(Node* node, string* err);
```

Comment:
Why?

Reply:
clang-tidy warns against that with readability-named-parameter

Reply:
Not every clang-tidy check is a good idea. I think readability-named-parameter should make an exception for this case (parameter name is lower-case version of type). The Google C++ Style Guide section this check is based also doesn't mention this explicitly.

Reply:
I'm very opposed to omitting the parameter name. I think that will result in substantially more confusion.